src/Magnum/Trade/PhongMaterialData.cpp

```diff
@@ -47,6 +47,17 @@ PhongMaterialData::PhongMaterialData(const Flags flags, const Color4& ambientCol
         _textureMatrix = textureMatrix;
 }
 
+PhongMaterialData::PhongMaterialData(Flags flags, const Color4& ambientColor, UnsignedInt ambientTexture, UnsignedInt ambientTextureLayer, const Color4& diffuseColor, UnsignedInt diffuseTexture, UnsignedInt diffuseTextureLayer, const Color4& specularColor, UnsignedInt specularTexture, UnsignedInt specularTextureLayer, UnsignedInt normalTexture, UnsignedInt normalTextureLayer, const Matrix3& textureMatrix, MaterialAlphaMode alphaMode, Float alphaMask, Float shininess, const void* importerState) noexcept: PhongMaterialData{flags, ambientColor, ambientTexture, diffuseColor, diffuseTexture, specularColor, specularTexture, normalTexture, textureMatrix, alphaMode, alphaMask, shininess, importerState} {
```

Comment:
I would switch the delegation order with the above constructor delegating into this one, supplying `0`s for the ~~layers~~ coordinate sets, and this constructor doing everything.